### PR TITLE
E_CLASSROOM-305 [FE] Reusable Input Component

### PR DESCRIPTION
**Pull request by ErickDoroy**
### Issue Link
- https://framgiaph.backlog.com/view/E_CLASSROOM-305
### Definition of Done
- [x] Input field is reusable
- [x] Existing components using should not be broken
- [x] Should also take into account a 'Search' input version
- [x] Different sizes should be taken into account

#### Main note
- http://localhost:3003/admin/theme
#### Pages Affected
- N/A

### Scenarios/ Test Cases
- N/A 
#### User Interface
- N/A
#### User Experience 
- N/A
#### Functionality
- N/A
### Screenshots
themeinput

### CODE REVIEW

**Comment by catampongan** [comment]
## Approved PR #149 
- [x] Different sizes are taken into account
- [x] Same design and sizes in Figma
--------------------------
task-305

--------------------------
- Total Test Time: 0.25 hr
- Total Iterations: 0
- Total Comments: 0

File: src/components/InputField/index.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+  > value        : Pass a string value input
+
+  > onChange     : Pass a string value.
+
+  > isInvalid    : Pass the error
+
+  > fieldSize    : Pass the size's of the input field by lg, md, and sm
+
+  > placeholder  : Pass a string value for the placeholder
```
Comment (karlolee50) [inline src/components/InputField/index.js]: please also put the possible values that can be passed as a prop

File: src/pages/admin/Theme/components/input.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+            <InputField
+              fieldSize="sm"
+              placeholder="Small Input"
+            />
+          </div>
+        </div>
+      </div>
+    </div>
+  );
+};
```
Comment (karlolee50) [inline src/pages/admin/Theme/components/input.js]: add extra line

**Review by karlolee50 — CHANGES_REQUESTED** [review]
Review time: 10 minutes
Comment/s: 2

File: src/components/InputField/index.module.scss [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    border: 2px $color-light-blue-1 solid;
+}
+
+.sm {
+    width: 200px;
+    height: 30px;
+    font-size: $font-size-md;
+    font-weight: $font-weight-regular;
+    border: 2px $color-light-blue-1 solid;
+}
```
Comment (karlolee50) [inline src/components/InputField/index.module.scss]: please update

File: src/components/InputField/index.js [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+  type,
+  value,
+  onChange,
+  fieldSize,
+  isInvalid,
+  placeholder,
+    
+}) => {
+
+  function styleSize() {
```
Comment (karlolee50) [inline src/components/InputField/index.js]: change into ES6 style
```suggestion
  const styleSize = () => {
```

**Review by karlolee50 — CHANGES_REQUESTED** [review]
Review time: 5 minutes
Comment/s: 2

**Review by karlolee50 — APPROVED** [review]

**Pull request merged**